pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -626,16 +635,25 @@ func (s *awsSdkEC2) DescribeVolumes(request *ec2.DescribeVolumesInput) ([]*ec2.V
 		}
 		request.NextToken = nextToken
 	}
-
+	timeTaken := time.Since(requestTime).Seconds()
+	recordAwsMetric("aws_describe_volume", timeTaken, "")
```

Comment:
Here it's only recording in the success case. I think it should be consist either way.